docs/content/contributing/tests.md

```diff
@@ -2,5 +2,15 @@
 title: 'Contributing | Tests'
 ---
 
-!!! todo
-    This section should provide a detailed step by step guide on how to write tests
\ No newline at end of file
+1. Install docker
+2. Execute `git submodule update --init --recursive`
+3. Install jq
+
+    !!! info "MacOS Specific (needed for tests)"
+        ```bash
+        brew install coreutils
+        # bash >= 4.0 for associative arrays
+        brew install bash
+        ```
```

Comment:
Formatting nit-pick, blank new line after admonition declaration (there's some cases where the parser otherwise breaks which breaks rendering, so it's best to try keep this convention for admonitions).

Feel free to use the commit suggestion button below this time :stuck_out_tongue: 

```suggestion
    !!! info "MacOS Specific (needed for tests)"

        ```bash
        brew install coreutils
        # bash >= 4.0 for associative arrays
        brew install bash
        ```
```